### ISSUE TITLE
Error reported for property overrides in RAML 1.0 Spec.

### ISSUE THREAD

**Issue by jacobstr** [mulesoft/api-workbench#69]
Using api-workbench 0.8.6.

My understanding is that property overrides are supported in 1.0. I've linked a pared-down example where workbench reports errors:

https://gist.github.com/jacobstr/8b1e0f9dc542e769dc4e

<img width="801" alt="screen shot 2015-12-02 at 11 52 50 am" src="https://cloud.githubusercontent.com/assets/70507/11542221/3c7fe214-98eb-11e5-952c-86fb9841837d.png">

**Comment by sichvoge** [comment]
maybe the confusion is the property `properties` as it is reserved?

**Comment by jacobstr** [comment]
I don't think that's the case - it's properly nested and things work without inheritance.

**Comment by sichvoge** [comment]
We found out the error I think. The Spec says:

> Subtypes can override properties defined in parent types. There are two restrictions: If a property is required in the parent, it cannot be made optional in the subtype The type of an already defined property can only be changed to a narrower type ( a type that specializes the parent type )

So a valid example would be something like this:

```yaml
#%RAML 1.0
title: Property override

types:
  FormattedAddress:
    type: string
    pattern: "someRegexpHere"

  Man:
    properties:
      address: string

  LawfulMan:
    type : Man
    properties:
      address: FormattedAddress
```

Still I think you found an error in the validation, since even with that the workbench complains ;)

**Comment by ddenisenko** [comment]
Fixed. Fix should be available in 0.8.10.
